src/DisplayModel.h

```diff
@@ -135,6 +135,8 @@ struct DisplayModel : public Controller {
     void Relayout(float zoomVirtual, int rotation);
 
     Rect GetViewPort() const;
+    bool hScrollbarShown() const;
```

Comment:
To be more in line with the rest of the code, this should be `IsHScrollbarVisible()`